src/content/chapter2_flow_control/lesson11_guards/en.html

```diff
@@ -4,6 +4,34 @@
   <code>True</code> for the pattern to match.
 </p>
 <p>
-  Only a limited set of operators can be used in guards, and functions cannot be
-  called at all.
-</p>
+  Only a limited set of operators can be used in guards:
+<ul>
+  <li><code>==</code> - equals</li>
+  <li><code>!=</code> - not equals</li>
+  <li><code>></code> - greater than (Int)</li>
+  <li><code>>=</code> - greater than or equal to (Int)</li>
+  <li><code><</code> - less than (Int)</li>
+  <li><code><=</code> - less than or equal to (Int)</li>
+  <li><code>>.</code> - greater than (Float)</li>
+  <li><code>>=.</code> - greater than or equal to (Float)</li>
+  <li><code><.</code> - less than (Float)</li>
+  <li><code><=.</code> - less than or equal to (Float)</li>
+  <li><code>+</code> - addition (Int)</li>
+  <li><code>+.</code> - addition (Float)</li>
+  <li><code>-</code> - subtraction (Int)</li>
+  <li><code>-.</code> - subtraction (Float)</li>
+  <li><code>*</code> - multiplication (Int)</li>
+  <li><code>*.</code> - multiplication (Float)</li>
+  <li><code>\</code> - division (Int)</li>
+  <li><code>\.</code> - division (Float)</li>
+  <li><code>%</code> - remainder (Int)</li>
+  <li><code>||</code> - Or</li>
+  <li><code>&&</code> - And</li>
+  <li><code>!</code> - Not</li>
+  <li><code>some_var</code> - variables</li>
+  <li><code>const_var</code> - constant</li>
```

Comment:
I opted not to include an entry for[1]
```rust
    ModuleSelect {
        location: SrcSpan,
        type_: Type,
        label: EcoString,
        module_name: EcoString,
        module_alias: EcoString,
        literal: Constant<Type, RecordTag>,
    },
```
assuming that for a user a local constant or one imported are conceptually the same.

[1]https://github.com/gleam-lang/gleam/blob/f9c2d98970241e51fba58cf998e10245f8a2fe05/compiler-core/src/ast.rs#L1522C1-L1529C7